src/wp-content/themes/twentytwentyone/functions.php

```diff
@@ -434,7 +434,7 @@ function twenty_twenty_one_scripts() {
 		null,
 		array(),
 		wp_get_theme()->get( 'Version' ),
-		true
+		array( 'in_footer' => true )
```

Comment:
Updated to reflect new function signature given prior specification of `$in_footer` param.